ingest/config/config.yaml

```diff
@@ -107,7 +101,7 @@ curate:
   abbr_authors_field: "abbr_authors"
   # Path to the manual annotations file
   # The path should be relative to the ingest directory
-  annotations: "defaults/annotations.tsv"
+  annotations: "config/annotations.tsv"
```

Comment:
Is there a specific reason for deviating from the [pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/tree/main/ingest/defaults) and choosing the `config` directory here? 

For context on why the `defaults` directory was chosen, see the commit: https://github.com/nextstrain/pathogen-repo-guide/commit/14e5dca9f8552c8427d7d4e360c964ca3ac1f8dd

I don't have a strong opinion on this matter other than maintaining consistency.

Reply:
because i used `config` in `seasonal-cov` ([e.g.](https://github.com/nextstrain/seasonal-cov/tree/main/ingest/config)) and nobody said anything about it, so i continued the pattern here >_<

i will align with the guide (and i will idly daydream about writing something to validate conformance to make this whole process a little more automatic...)

Reply:
> (and i will idly daydream about writing something to validate conformance to make this whole process a little more automatic...)

(Totally welcome to take on https://github.com/nextstrain/pathogen-repo-guide/issues/28 😄)

Reply:
switched to `data/`